src/gtk/dc.cpp

```diff
@@ -350,23 +364,29 @@ wxScreenDCImpl::wxScreenDCImpl(wxScreenDC* owner)
     gc->EnableOffset(m_contentScaleFactor <= 1);
     SetGraphicsContext(gc);
 }
+
+wxSize wxScreenDCImpl::GetPPI() const
```

Comment:
`wxScreenDC` doesn't work with GTK+3, so I have to wonder if it's worth adding this.

Reply:
Yes, I thought it didn't too, but there is non-trivial code in its ctor, so I decided that I was remembering wrong...

But actually, I think `GetPPI()` is one of the few methods which it might still make sense to provide even if drawing on it doesn't work (the other one could be `GetSize()`).